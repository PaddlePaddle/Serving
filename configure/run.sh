for i in FileReloader.cpp ConfigReloader.cpp CmdOption.cpp ConfigError.cpp ConfigGroup.cpp ConfigUnit.cpp Configure.cpp constraint/Constraint.cpp constraint/ConstraintFunction.cpp constraint/cc_default.cpp reader/RawData.cpp reader/Reader.cpp utils/init.cpp utils/trans.cpp cfgext.cpp cfgflag.cpp compiler/src/idl.c compiler/src/idl_lex.c compiler/src/idl_gram.c ; do echo -e "\${CMAKE_CURRENT_LIST_DIR}/$i"; done

