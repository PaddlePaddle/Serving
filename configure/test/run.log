WARNING: 02-13 23:41:42:====RUN ./test_demo====
====RUN ./test_err====
====RUN ./test_get_value====
====RUN ./test_option====
====RUN ./test_rawdata====
====RUN ./test_reader====
====RUN ./test_rebuild====
====RUN ./test_str====
====RUN ./test_t1====
====RUN ./test_trans_float====
====RUN ./test_trans_str2str====
====RUN ./test_xiaowei====
======= Done! ==============

[1;32mIf *.log is diff to *.res, the diff is written to [31mres[m

9023 (%&^%&&^¹þ¹þ
WARNING: 02-13 23:41:42:   * 0 Reader configure file [.//new.conf] error. Stop.
WARNING: 02-13 23:41:42:   * 0 Config : Get [SIZE] : out of range.
WARNING: 02-13 23:41:42:   * 0 Config error: Max $include level is 1 [./inc.conf:4]
WARNING: 02-13 23:41:42:   * 0 Configure : Section Error [....Wrong] Ignored.
WARNING: 02-13 23:41:42:   * 0 Error [File:./new.conf, Line:27] [....Wrong]
WARNING: 02-13 23:41:42:   * 0 Configure : Section Error [...XXX] Ignored.
WARNING: 02-13 23:41:42:   * 0 Error [File:./new.conf, Line:69] [...XXX]
WARNING: 02-13 23:41:42:   * 0 Unsupport _key format [*(&(*]
WARNING: 02-13 23:41:42:   * 0 Error [File:./new.conf, Line:70] *(&(*:asf 239023 (%&^%&&^¹þ¹þ
WARNING: 02-13 23:41:42:   * 0 Reader configure file [./new.conf] error. Stop.
WARNING: 02-13 23:41:42:   * 0 Constraint : Key not in range(), in [File:NULL Line:0 Key:Putian Value:100] : [File:t1.cons Line:8]=[Putian]
WARNING: 02-13 23:41:42:   * 0 Constraint : Key not in range(), in [File:NULL Line:0 Key:@num Value:11] : [File:t1.cons Line:9]=[num]
WARNING: 02-13 23:41:42:   * 0 Constraint : Key not in range(), in [File:NULL Line:0 Key:@num Value:-2] : [File:t1.cons Line:9]=[num]
WARNING: 02-13 23:41:42:   * 0 Constraint : array has too many elements! [File:t1.cons Line:10]=[ip]
WARNING: 02-13 23:41:42:   * 0 Constraint : Key is not ip(), in [File:NULL Line:0 Key:@ip Value:123.-1.2.6] : [File:t1.cons Line:10]=[ip]
WARNING: 02-13 23:41:42:   * 0 Constraint : key not match regexp(). [File:NULL Line:0 Key:@reg Value:S2sadf] : [File:t1.cons Line:12]=[reg]
WARNING: 02-13 23:41:42:   * 0 Constraint : length() : value is too long.[File:NULL Line:0 Key:@reg Value:asdlLLLkdkksdf] : [File:t1.cons Line:12]=[reg]
