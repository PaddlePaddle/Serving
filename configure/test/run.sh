
echo ====RUN ./test_conf====
./test_conf > ./test_conf.log
diff ./test_conf.res ./test_conf.log >> res


echo ====RUN ./test_demo====
./test_demo > ./test_demo.log
diff ./test_demo.res ./test_demo.log >> res


echo ====RUN ./test_err====
./test_err > ./test_err.log
diff ./test_err.res ./test_err.log >> res


echo ====RUN ./test_get_value====
./test_get_value > ./test_get_value.log
diff ./test_get_value.res ./test_get_value.log >> res


echo ====RUN ./test_option====
./test_option > ./test_option.log
diff ./test_option.res ./test_option.log >> res


echo ====RUN ./test_rawdata====
./test_rawdata > ./test_rawdata.log
diff ./test_rawdata.res ./test_rawdata.log >> res


echo ====RUN ./test_reader====
./test_reader > ./test_reader.log
diff ./test_reader.res ./test_reader.log >> res


echo ====RUN ./test_rebuild====
./test_rebuild > ./test_rebuild.log
diff ./test_rebuild.res ./test_rebuild.log >> res


echo ====RUN ./test_str====
./test_str > ./test_str.log
diff ./test_str.res ./test_str.log >> res


echo ====RUN ./test_t1====
./test_t1 > ./test_t1.log
diff ./test_t1.res ./test_t1.log >> res


echo ====RUN ./test_trans_float====
./test_trans_float > ./test_trans_float.log
diff ./test_trans_float.res ./test_trans_float.log >> res


echo ====RUN ./test_trans_str2str====
./test_trans_str2str > ./test_trans_str2str.log
diff ./test_trans_str2str.res ./test_trans_str2str.log >> res


echo ====RUN ./test_xiaowei====
./test_xiaowei > ./test_xiaowei.log
diff ./test_xiaowei.res ./test_xiaowei.log >> res

echo ======= Done! ==============
echo 
echo | awk '{print "\033[1;32mIf *.log is diff to *.res, the diff is written to \033[31mres\033[m"}'
echo 
