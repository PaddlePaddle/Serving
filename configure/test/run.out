
=====test_conf=====
read = [FILE]
No such key
read = [0]

=====test_demo=====
Module Name : Config Demo
We have 2 servers.
Server[0] is : ip = 127.0.0.1, port = 9900
Server[1] is : ip = 192.168.1.2, port = 9901

=====test_err=====
No such _key
Format error

=====test_get_value=====

=====test_option=====

=====test_rawdata=====
#进程名
COMLOG_PROCNAME : test_others
COMLOG_LEVEL : 16
#支持的设备数目
COMLOG_DEVICE_NUM : 5
#支持的自定义等级
COMLOG_SELFDEFINE : xiaowei, hello
#支持的设备名
COMLOG_DEVICE0 : FILE
COMLOG_DEVICE1 : AFILE
COMLOG_DEVICE2 : NET
COMLOG_DEVICE3 : ULLOG
COMLOG_DEVICE4 : ULNET

#device 0: FILE
FILE_TYPE : FILE
FILE_NAME : test_others_file.log
FILE_PATH : ./log
FILE_SYSLEVEL : 16
FILE_OPEN : 1
FILE_SELFLEVEL : xiaowei, hello
FILE_SIZE : 2000

#device 1: AFILE
AFILE_TYPE : AFILE
AFILE_NAME : test_others_afile.log
AFILE_PATH : ./log
AFILE_SYSLEVEL : 16
AFILE_OPEN : 1
AFILE_SELFLEVEL : xiaowei, hello
AFILE_SIZE : 2000

#device 2: NET
NET_TYPE : NET
NET_NAME : test_others_net.log
NET_PATH : 127.0.0.1:9898/data
NET_OPEN : 1
NET_SYSLEVEL : 16
NET_SELFLEVEL : xiaowei, hello

#device 3: ULLOG
ULLOG_TYPE : ULLOG
ULLOG_NAME : test_others_ullog.log
ULLOG_PATH : ./log
ULLOG_SYSLEVEL : 16
ULLOG_OPEN : 1
ULLOG_SELFLEVEL : xiaowei, hello
ULLOG_SIZE : 2000

#device 4: ULNET
ULNET_TYPE : ULNET
ULNET_NAME : test_others_ulnet.log
ULNET_PATH : 127.0.0.1:9898/log
ULNET_OPEN : 1
ULNET_SYSLEVEL : 16
ULNET_SELFLEVEL : xiaowei, hello

=====test_reader=====

=====test_rebuild=====
key: 10086
key: 10086

=====test_str=====
str1 : to raw : "xyz\x41\x42\x43"
str1 : to std : xyzABC
str1 : to cstr: xyzABC
str2 : to raw : x234567890y234567890
str2 : to std : x234567890y234567890
str2 : to cstr: x234567890y2345

=====test_t1=====

=====test_trans_float=====
[OK] check [3.1415926] IS a float[3.14159]
[OK] check [3.1415926e100] IS a float[3.14159e+100]
[OK] check [3.1415926E-10] IS a float[3.14159e-10]
[OK] check [3.1415926e] ISNOT a float[0]
[OK] check [.1415926] IS a float[0.141593]
[OK] check [-.1415926] IS a float[-0.141593]
[OK] check [3.E] ISNOT a float[0]
[OK] check [+-3.1] ISNOT a float[0]
[OK] check [e103] ISNOT a float[0]
[OK] check [3.1415926e1.1] ISNOT a float[0]
[OK] check [3.1415926e-10] IS a float[3.14159e-10]
[OK] check [3.1415926E+0997] IS a float[inf]
[OK] check [-.141e5926] IS a float[-inf]
[OK] check [3+.1415926] ISNOT a float[0]

=====test_trans_str2str=====
----- Read -----
"test"
----- trans to -----
test
------ Finish -----
----- Read -----
"Haha"ddd
----- trans to -----
Haha
------ Finish -----
----- Read -----
"\tThis is \"Hawk\".\n"sadf
----- trans to -----
	This is "Hawk".

------ Finish -----
----- Read -----
askdfj;aj jaskdfj;askf
----- trans to -----
askdfj;aj jaskdfj;askf
------ Finish -----
----- Read -----
"\x41---"
----- trans to -----
A---
------ Finish -----
----- Read -----
"\101---"
----- trans to -----
A---
------ Finish -----

=====test_xiaowei=====
Section-------[GLOBAL]
    Array--------[server] : {
        Section-------[GLOBAL.@server]
            =[ip], _value=[127.0.0.1]
            =[port], _value=[5123]
        Section-------[GLOBAL.@server]
            =[ip], _value=[127.0.0.2]
            =[port], _value=[1123]
        Section-------[GLOBAL.@server]
            =[ip], _value=[138.3.1.2]
            =[port], _value=[1234]
    }
    Section-------[GLOBAL.person]
        =[name], _value=[xiaowei]
        =[age], _value=[1]
conf size = 2
127.0.0.1
5123
127.0.0.2
1123
138.3.1.2
1234
-------catch NoSuchKey---------
